std_msgs/Header header
int16 ls_var1
int16 ls_var2
int16 ls_var3
